manifests/v2.1.0/vsphere-67u3/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -0,0 +1,136 @@
+# Minimum Kubernetes version - 1.16
```

Comment:
Can you change this to v1.17.0?
As per - https://vsphere-csi-driver.sigs.k8s.io/releases/v2.1.0.html

Reply:
Done.